Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
+        {
+            string value = null;
+            if (progInfo != null)
+            {
+                value = GetValue(name + progInfo.Progid.ToString(CultureInfo.InvariantCulture));
```

Comment:
Adding a `progid` column in the settings table instead of appending it to the setting name would probably make some of the actions in the GUI easier when you get to that (e.g. setting values for all programmes back to a single value).